plugins/modules/proxysql_backend_servers.py

```diff
@@ -413,32 +415,28 @@ def delete_server(self, check_mode, result, cursor):
 
 def main():
     module = AnsibleModule(
-        argument_spec=dict(
-            login_user=dict(default=None, type='str'),
-            login_password=dict(default=None, no_log=True, type='str'),
-            login_host=dict(default='127.0.0.1'),
-            login_unix_socket=dict(default=None),
-            login_port=dict(default=6032, type='int'),
-            config_file=dict(default='', type='path'),
-            hostgroup_id=dict(default=0, type='int'),
-            hostname=dict(required=True, type='str'),
-            port=dict(default=3306, type='int'),
-            status=dict(choices=['ONLINE',
-                                 'OFFLINE_SOFT',
-                                 'OFFLINE_HARD']),
-            weight=dict(type='int'),
-            compression=dict(type='int'),
-            max_connections=dict(type='int'),
-            max_replication_lag=dict(type='int'),
-            use_ssl=dict(type='bool'),
-            max_latency_ms=dict(type='int'),
-            comment=dict(default='', type='str'),
-            state=dict(default='present', choices=['present',
-                                                   'absent']),
-            save_to_disk=dict(default=True, type='bool'),
-            load_to_runtime=dict(default=True, type='bool')
-        ),
-        supports_check_mode=True
+        supports_check_mode=True,
+        argument_spec=proxysql_common_argument_spec(
+            dict(
+                hostgroup_id=dict(default=0, type='int'),
+                hostname=dict(required=True, type='str'),
+                port=dict(default=3306, type='int'),
+                status=dict(choices=['ONLINE',
+                                     'OFFLINE_SOFT',
+                                     'OFFLINE_HARD']),
+                weight=dict(type='int'),
+                compression=dict(type='int'),
+                max_connections=dict(type='int'),
+                max_replication_lag=dict(type='int'),
+                use_ssl=dict(type='bool'),
+                max_latency_ms=dict(type='int'),
+                comment=dict(default='', type='str'),
+                state=dict(default='present', choices=['present',
+                                                       'absent']),
+                save_to_disk=dict(default=True, type='bool'),
+                load_to_runtime=dict(default=True, type='bool')
+            )
+        )
```

Comment:
How about the approach from, say, community.postgresql?
```
    argument_spec = postgres_common_argument_spec()
    argument_spec.update(
        db=dict(type="str", required=True, aliases=["login_db"]),
        ext=dict(type="str", required=True, aliases=["name"]),
        schema=dict(type="str"),
        state=dict(type="str", default="present", choices=["absent", "present"]),
        cascade=dict(type="bool", default=False),
        session_role=dict(type="str"),
        version=dict(type="str"),
        trust_input=dict(type="bool", default=True),
    )
```
It's, at least, allows to avoid deep indentation